build/lib/development-keys-path.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+const join = require('path').join;
+module.exports = join(process.cwd(), '.env');
```

Comment:
I wonder if this can be moved into the file below? Seems a little lonely here, and it's only used in that one place...

Reply:
(Discussed in real life) We think we can actually remove most of this `.env` stuff. It's only required for running `nui rebuild` locally, which isn't very often. And can be achieved by doing `export CIRCLE_REBUILD_KEY=blah; nui rebuild`. So we can put that in the README and remove all the code.